connectors/katalog/pkg/connector/utils/configuration.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 IBM Corp.
```

Comment:
Can this TLS configuration be shared for all connectors ?

Reply:
Yes, I guess so. regarding where to put the code - `pkg/tls` is also shared by the manager so I might need to change the naming of the secrets to make them suitable for the client and secret to use.